sections/projectstructre/separateexpress.md

```diff
@@ -8,46 +8,58 @@ The latest Express generator comes with a great practice that is worth to keep -
 
 <br/><br/>
 
-### Code example: API declaration, should reside in app.js
+### Code example: API declaration, should reside in app.js/app.ts
 
 ```javascript
-var app = express();
+const app = express();
 app.use(bodyParser.json());
 app.use("/api/events", events.API);
 app.use("/api/forms", forms);
 ```
 
-<br/><br/>
-
 ### Code example: Server network declaration, should reside in /bin/www
 
-```javascript
+<details>
+<summary><strong>Javascript</strong></summary>
+<pre lang="javascript">
 var app = require('../app');
 var http = require('http');
-
-/**
- * Get port from environment and store in Express.
- */
-
+<br/>
+// Get port from environment and store in Express.

```

Comment:
I had to change the comment style to avoid weird display bugs. `*` were interpreted as markdown bullet points